### ISSUE TITLE
Minor feedback on new documentation

### ISSUE THREAD

**Issue by jcoulter12** [ianhi/mpl-interactions#69]
Hi @ianhi and collaborators (possibly @samanthahamilton), 

I installed the code went through the documentation (from the ReadTheDocs page and README file). As I have a fresh set of eyes and this is my first time looking at the package, I had a couple thoughts about the docs I wanted to share in case it's helpful to get some "new user" feedback. 

Looks great so far! I'm happy to clarify any thoughts or test any new examples later down the road, too.
Jenny

#### README
* Can you put a one liner describing what the package is/does, and a link to the ReadTheDocs directly at the top of the README? I felt like this could be helpful for someone who just stumbled onto the package and needed the quick and dirty on what it does. Maybe move the line "This library provides helpful ways to interact with Matplotlib plots."  and make that into a summary at the very top, and keep it separate from the mention of the submodules? 
* Also, if there are ever examples of jupyter notebooks that run the (IDK if there are) it would be nice to indicate where those are right at the top, so someone could get a fast look at what the package does. 
* Maybe also an overall comment -- I love the use of gifs in the README, and overall README feels really good. It seems though that maybe the README contains information that is not in the full Docs page (like concerning the three modules), and that the README should sort of be the quick rundown of the crucial parts of the docs, like basic install and overview. Right now, Docs and README feel kind of out of sync. 

#### Installation
* Are the installation instructions in the README synced with the ones in read the docs? I think the `conda install nodejs=12` line is a little different from the one in README, might want to be consistent. 
* Very silly, but for people who don't know much about Jupyterlab (like me), but are clearly intending to use this in a notebook, maybe rephrase slightly the first line in [jupyterlab setup section](https://mpl-interactions.readthedocs.io/en/latest/Installation.html#jupyterlab-setup)  "If you plan on using mpl_interactions in the notebook..." to something like "If you plan on using mpl_interactions in a Jupyterlab notebook rather than a traditional jupyter notebook". It's not critical, but the wording made take a minute to figure out if I needed to take this step, when I actually didn't. 

#### Examples section of ReadTheDocs
* In [simple example](https://mpl-interactions.readthedocs.io/en/latest/examples/jupyter-notebooks.html#simple-example) did you want tau and beta to be spelled out in one code block, but written as greek letters in the second one? Naively pasting this caused an error for me, but maybe I'm not using some unicode setting.

**Comment by samanthahamilton** [comment]
Hi, @jcoulter12,

Thanks for the feedback about the docs layout and presentation. You're right to say "a set of fresh eyes" is helpful. I've jumped in to make some of the general changes you suggested. See below:

>Can you put a one liner describing what the package is/does, and a link to the ReadTheDocs directly at the top of the README?

Great idea, I've rearranged the opening lines to add a quick summary and link. 

>maybe the README contains information that is not in the full Docs page

I agree on this---the Read the Docs page has come together very quickly and recently, and we are working to update and sync!

>I think the conda install nodejs=12 line is a little different from the one in README

Thanks for pointing that out.

I've made some notes for future adjustments. Thanks again, Jenny.
Cheers,
Sam.

**Comment by jcoulter12** [comment]
Hi @samanthahamilton, 

Thanks for considering my feedback and making changes. Again, the overall the docs look great and definitely were enough for me to figure out how to use the code. Keep up the good work, and if there's ever any overhaul of the documentation, tutorials, or code functionality, you or @ianhi can feel free to tap me to go through it again. 

-- Jenny

**Comment by ianhi** [comment]
Thanks for the notes jenny!

@all-contributors please add @jcoulter12 for userTesting
